eval/use_eval.ts

```diff
@@ -0,0 +1,130 @@
+import type { Context, Denops, Dispatcher, Meta } from "@denops/core";
+import { isString } from "@core/unknownutil/is/string";
+import { isUndefined } from "@core/unknownutil/is/undefined";
+import { ulid } from "@std/ulid";
+import { execute } from "../helper/execute.ts";
+import { stringify } from "./stringify.ts";
+
+// Note: Imports only types and is used only in tsdoc.
+// deno-lint-ignore no-unused-vars
+import type { Expression } from "./expression.ts";
+// deno-lint-ignore no-unused-vars
+import type { RawString } from "./string.ts";
```

Comment:
Why do we need it?

Reply:
https://github.com/vim-denops/deno-denops-std/pull/260#discussion_r1718460996